apollo-portal/src/main/resources/static/i18n/zh-CN.json

```diff
@@ -856,5 +856,33 @@
   "Rollback.RollbackSuccessfully": "回滚成功",
   "Rollback.RollbackFailed": "回滚失败",
   "Revoke.RevokeFailed": "撤销失败",
-  "Revoke.RevokeSuccessfully": "撤销成功"
+  "Revoke.RevokeSuccessfully": "撤销成功",
+  "ApolloAuditLog.Title": "审计日志",
+  "ApolloAuditLog.TraceAuditLogTips": "链路包含审计日志",
+  "ApolloAuditLog.RelatedDataInfluenceTips": "审计日志相关数据变动",
+  "ApolloAuditLog.DataInfluenceTips": "数据变动展示",
+  "ApolloAuditLog.DataInfluence.EntityName": "实体名称",
+  "ApolloAuditLog.DataInfluence.EntityId": "实体ID",
+  "ApolloAuditLog.DataInfluence.FieldName": "实体属性名",
+  "ApolloAuditLog.DataInfluence.FieldNewValue": "修改属性值",
+  "ApolloAuditLog.DataInfluence.HappenedTime": "发生时间",
+  "ApolloAuditLog.TraceIdTips": "链路唯一ID",
+  "ApolloAuditLog.OpName": "操作名称",
+  "ApolloAuditLog.HappenedTime": "发生时间",
+  "ApolloAuditLog.StartDate": "开始时间",
+  "ApolloAuditLog.QueryTips": "查询审计日志清单",
```

Comment:
use ApolloAuditLog.Title instead of ApolloAuditLog.QueryTips

Reply:
changed~